src/components/Accordion/Accordion.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="sb-accordion" :class="computedClass">
+    <button
+      class="sb-accordion__button"
+      :aria-expanded="isOpenLocal"
+      :aria-controls="`collapse${_uid}`"
+      @click="toggleAccordion"
+    >
+      <SbIcon :name="chevronIcon" />
+      <p class="sb-accordion__title">{{ title }}</p>
+
+      <SbIcon
+        v-if="icon"
+        v-tooltip="{ label: iconDescription }"
+        :name="icon"
+        class="sb-accordion__icon"
+        @click="$emit('icon-click')"
```

Comment:
I think, you could change to `@click.stop="$emit('icon-click')"` to not open or close the component when click on this button, what do you think ?

Reply:
Good idea